client/packages/system/src/Patient/DetailModal/DetailModal.tsx

```diff
@@ -80,36 +69,6 @@ export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
               }
             />
           </DetailSection>
-          <DetailSection title="">
-            <DetailInputWithLabelRow
-              label={t('label.date-created')}
-              inputProps={{
-                value: data?.createdDatetime
-                  ? localisedDate(data?.createdDatetime)
-                  : '',
-                disabled: isDisabled,
-              }}
-            />
-            <DetailInputWithLabelRow
-              label={t('label.manufacturer')}
-              Input={
-                <Checkbox
-                  disabled={isDisabled}
-                  checked={data?.isManufacturer}
-                />
-              }
-            />
-            <DetailInputWithLabelRow
-              label={t('label.donor')}
-              Input={<Checkbox disabled={isDisabled} checked={data?.isDonor} />}
-            />
-            <DetailInputWithLabelRow
-              label={t('label.on-hold')}
-              Input={
-                <Checkbox disabled={isDisabled} checked={data?.isOnHold} />
```

Comment:
Removed elements no longer part of "patients" endpoint fields

Reply:
`chargeCode` might be required - an mSupply person would have to help us with that.
The date created seems useful to have available in the API - can we add that in?

Reply:
I'll ask @clemens-msupply to include it. But this branch is now in sync with the feature branch schema-wise, so I'll merge this and I'll add it back in when the back-end does.